.eslintrc

```diff
@@ -7,7 +7,6 @@
     "no-underscore-dangle": 0,
     "no-use-before-define": [2, "nofunc"],
     "max-len": [2, 150, 2],
-    "quotes": [2, "single"],
```

Comment:
Why this change? 👀 It generates a lot of noise in this PR 😕

Reply:
My prettier modifie all files with " " instead of ' ' and
I can't do this :

"quotes": [2, "single", "double"]
So I removed it instead of revert all prettier modifications

Reply:
I just saw your comment above but it is not a good reason to change lint rules in this project 😄 You should find why your prettier is misconfigured. Maybe a `.prettierrc` is missing here.

Reply:
Yes it was planned but you are too fast for review 😅